add a new member

starbuckscoder opened the pull request.

wschae commented on app.js:
```diff
@@ -43,6 +43,7 @@ router.get('/people', function(req, res) {
         {name: 'swparkz', url: 'assets/swparkz.png'},
         {name: 'gsongsong', url: 'assets/sjeon.jpg'},
         {name: 'tak0ya', url: 'assets/tak0ya.jpg'}
```
Can you add a comma at the end of Line 45?

starbuckscoder replied:
Dear Professor Chae,

I checked my code is working from localhost. 
And I add and commit as explained on the course website. 

It goes well except at push level with this comment.

"Permission to sein92/projectx.git denied to starbuckscoder"

I ignored it and I moved forward by requesting merge. Now I manually "added a comma, added my photo to the file"
Could you cast a light on my procedure?

wschae replied:
One difference I noticed was we were supposed to clone from "wochae/projectx.git" instead of "sein92/projectx.git". Other than this, I could not see what went wrong. One thing I can recommend was to use "gitk" or similar tools to review the changes before push :)

wschae commented on app.js:
```diff
@@ -43,6 +43,7 @@ router.get('/people', function(req, res) {
         {name: 'swparkz', url: 'assets/swparkz.png'},
         {name: 'gsongsong', url: 'assets/sjeon.jpg'},
         {name: 'tak0ya', url: 'assets/tak0ya.jpg'}
+        {name: 'kkim', url: 'assets/kkim.jpg'}
```
I cannot see this file added into "assets" folder. Can you add the file there?

wschae commented:
You can ignore "The Travis CI build failed" for now. I will fix it later :)

wschae commented:
👍

wschae commented:
![image](https://user-images.githubusercontent.com/4300700/31646639-d77d8dec-b2b7-11e7-93eb-54d67c58f2db.png)

starbuckscoder commented:
Thank you for your approval. However, I am still wondering whether my procedure was expected process. Seems like add files/change code manually is not expected one....

starbuckscoder commented:
I left a comment over the upper dialog and just copied here for your opinion...
"Dear Professor Chae,

I checked my code is working from localhost.
 And I add and commit as explained on the course website.

It goes well except at push level with this comment.

"Permission to sein92/projectx.git denied to starbuckscoder"

I ignored it and I moved forward by requesting merge. Now I manually "added a comma, added my photo to the file"
 Could you cast a light on my procedure?"

Thank you again for kind consideration.

wschae commented:
We can try another push to check what went wrong. First, make sure sync/pull all changes from wochae/projectx. Then, add a new profile. then, git add. git commit. Then, use "gitk" or any similar tools to review what has been committed. Then, push to see if the result remains same or not :)

starbuckscoder commented:
oh, maybe I was syncing from the sein92/ rather than wschae/, which connect me to sein92 rather than you.
Thank you!
